plugin/config.json

```diff
@@ -0,0 +1,43 @@
+{
+	"Description": "VMWare vSphere Docker Volume plugin",
+	"Documentation": "http://vmware.github.io/docker-volume-vsphere/documentation",
+	"Entrypoint": ["/usr/bin/docker-volume-vsphere", "--log_level", "info"],
+	"PropagatedMount": "/mnt/vmdk",
+	"Mounts": [
```

Comment:
yeah. I did not want to open up all important folders. I think I will change it to /etc/vmware/<config> and open /etc/vmware. Comments ?